src/FondOfSpryker/Glue/CompanyUsersRestApi/Controller/CompanyUsersResourceController.php

```diff
@@ -33,6 +48,19 @@ public function getAction(RestRequestInterface $restRequest): RestResponseInterf
     }
 
     /**
+     * @Glue({
+     *     "post": {
+     *          "summary": [
+     *              "Create a company user"
+     *          ],
+     *          "parameters": [{
+     *              "ref": "acceptLanguage"
+     *          }],
+     *          "responses": {
```

Comment:
No response?

Reply:
@DerCalli it will be added, first i need to refractor the other Classes